pkg/storage/wrapper/base58/base58wrapper.go

```diff
@@ -3,27 +3,27 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package base58wrapper
+package base58
 
 import (
 	"github.com/btcsuite/btcutil/base58"
 
 	"github.com/hyperledger/aries-framework-go/pkg/storage"
 )
 
-// package base58wrapper offers provider.Store wrappers for stores with certain constraints on key values such as
+// package base58 offers provider.Store wrappers for stores with certain constraints on key values such as
 // reserved prefix values (eg key IDs starting with underscore) by wrapping key values in another format supported by
 // the store.
 
-// NewBase58StoreWrapper creates a new Base58StoreWrapper of store.
+// NewBase58StoreWrapper creates a new StoreBase58Wrapper of store.
 func NewBase58StoreWrapper(store storage.Store) storage.Store {
-	return &Base58StoreWrapper{store: store}
+	return &StoreBase58Wrapper{store: store}
 }
 
-// Base58StoreWrapper is a wrapper store that converts key IDs from base64 to base58 encoded values.
+// StoreBase58Wrapper is a wrapper store that converts key IDs from base64 to base58 encoded values.
```

Comment:
if there is no usage for this package I would have deleted it. It can be done is separate PR.

Reply:
there is no usage of this wrapper, I will remove it

Reply:
done